### DISCUSSION TITLE
HP Elitedesk 800 G2 - PFSENSE+Security Onion - Worth it?

### DISCUSSION THREAD

**Discussion by elliottmotson** [Security-Onion-Solutions/securityonion#6423] [Unsupported Versions] [answered]
Hi all, I'm new to Security Onion and have been running it in a virtual environment on my rack for a little while but would like to save energy and build a for-purpose, low power box to use Suricata and Zeek alongside PFSENSE to replace my current router. 

Specs of the machine would be as follows:

16GB DDR4 
i5-6500T 4c4t
And a 1TB nvme I have spare

Max throughput inbound is 350Mbps
For the PFSENSE side of things i've looked at getting a small 6 port Mikrotik smart switch.

If anyone has any pointers or suggestions i'd love to hear it :)

**Comment by tomjones1977** [comment, marked as answer]
This feels like the bare minimum you'd need to run SO, let alone pfsense.  You'd be better off running pfSense seperately especially if it's running your main internet connection.  You can still run Suricata within pfSense if you can't mirror the network traffic and pass the logs to SO.
